packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -360,6 +360,11 @@ export interface DashboardLayoutProps {
     toolbarActions?: {};
     toolbarAccount?: AccountProps;
   };
+  /**
+  * If the navigation bar and button should to be hidden
```

Comment:
Can we rephrase it to `Whether the navigation bar and menu icon should be hidden`?